lib/Expression/ColorMap/SolarisedColorMap.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace PhpBench\Expression\ColorMap;
+
+use Closure;
+use PhpBench\Expression\Ast\ArithmeticOperatorNode;
+use PhpBench\Expression\Ast\BooleanNode;
+use PhpBench\Expression\Ast\ComparisonNode;
+use PhpBench\Expression\Ast\DisplayAsNode;
+use PhpBench\Expression\Ast\FunctionNode;
+use PhpBench\Expression\Ast\Node;
+use PhpBench\Expression\Ast\NumberNode;
+use PhpBench\Expression\Ast\ParameterNode;
+use PhpBench\Expression\Ast\ParenthesisNode;
+use PhpBench\Expression\Ast\PercentDifferenceNode;
+use PhpBench\Expression\Ast\StringNode;
+use PhpBench\Expression\Ast\ToleratedTrue;
+use PhpBench\Expression\ColorMap;
+use PhpBench\Util\Gradient;
+
+/**
+ * @implements ColorMap<Node>
+ */
+class SolarisedColorMap implements ColorMap
```

Comment:
Using the Solarized theme doesn't make much sense unless it's applied to all styles in PHPBench.

This can be a 256 color theme, using mostly the standard 8 bit palette with gradients and more subtle colors of  time units etc.